### PR TITLE
rest-api-client

### PR DESCRIPTION
**Pull request by vasap87**
rest-api-client and changed rest-api-server(added stat path, modify method getDiffVUFS)

### CODE REVIEW

**Comment by vasap87** [comment]
In rest-api-client module interesting classes: com.stnetix.ariaddna.client.ApiClient and com.stnetix.ariaddna.client.api.DefaultApi

File: rest-api-server/src/gen/java/io/swagger/api/ApiException.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package io.swagger.api;
 
-@javax.annotation.Generated(value = "class io.swagger.codegen.languages.JavaJerseyServerCodegen", date = "2016-12-26T14:00:48.377Z")
+@javax.annotation.Generated(value = "io.swagger.codegen.languages.JavaJerseyServerCodegen", date = "2017-02-08T08:47:23.330Z")
 public class ApiException extends Exception{
```
Comment (kosmoflyko) [inline rest-api-server/src/gen/java/io/swagger/api/ApiException.java]: extend ArriaddnaException, see from above

File: rest-api-client/README.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+import io.swagger.client.api.DefaultApi;
+
+import java.io.File;
+import java.util.*;
+
+public class DefaultApiExample {
+
+    public static void main(String[] args) {
+        
+        DefaultApi apiInstance = new DefaultApi();
```
Comment (vasap87) [inline rest-api-client/README.md]: Yes, we can. But after regenerate this API change will be lost.

**Comment by kosmoflyko** [comment]
fix #17

**Pull request merged**